docs/source/zero-code-change-limitations.rst

```diff
@@ -160,21 +219,83 @@ the behavior of ``max_leaf_nodes``. The cuML RF treats this parameter as a "soft
 ``sklearn.kernel_ridge.KernelRidge``
```

Comment:
We should probably remove KernelRidge entirely, but we can do that in a follow-up.